examples/test-app/app.js

```diff
@@ -20,12 +20,23 @@ const baseUrl = `http://localhost:${config.port}`;
 
 lookAndFeel.configure(app, {
   baseUrl,
-  express: { views: [path.resolve(__dirname, 'steps')] },
+  express: {
+    views: [
+      path.resolve(__dirname, 'steps'),
+      path.resolve(__dirname, 'views')
```

Comment:
is this related to this pull request?

Reply:
Need to add that folder otherwise you can't do `{% extend "template.html" %}. Nunjucks is using expresses view folders as the files that are in scope for extends / imports / etc